CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.7.6
+
+BUGFIXES:
+* resource/signalfx_data_link: Handle extraneous empty targets [#309](https://github.com/splunk-terraform/terraform-provider-signalfx/pull/309)
+* resource/signalfx_data_link: Skip invalid target in data links [#306](https://github.com/splunk-terraform/terraform-provider-signalfx/pull/306)
```

Comment:
Is this description accurate for PR #306 ?

Reply:
ah yeah I must have pulled that from one of the commits from the PR instead of the overall PR. Fixed, thanks!